README.md

```diff
@@ -44,6 +44,8 @@ rails app:template LOCATION=https://raw.githubusercontent.com/nimblehq/rails-tem
 
 Available Addons:
 - `docker`
+- `semaphore`
+- `bootstrap`
```

Comment:
hi guys, now we have more addon, should we categorize them? 🤔 
```text
- Frontend:
- CI/CD:
```

Reply:
I think we can't separate them via category for now because the ADDON list there is support for this example

![Pasted_Image_10_14_20__13_47](https://user-images.githubusercontent.com/11751745/95953363-e72c0c80-0e23-11eb-89d5-4e10fcb1bd59.png)

If we separate it per category, we need to adjust the README to make it clear.

Reply:
IMO For updating the README content, we can do that on a separate PR @byhbt @rafayet-monon as it's out of this PR scope, this PR is focusing on adding Boostrap addon, what do you think?

Reply:
yes, I meant to update the README content, anyway, we should do it another PR. That's fine now.